packages/mds-jurisdiction-service/service/handlers/create-jurisdiction-handler.ts

```diff
@@ -15,13 +15,22 @@
  */
 
 import { ServiceResponse, ServiceError, ServiceResult } from '@mds-core/mds-service-helpers'
-import { ValidationError, ConflictError, ServerError } from '@mds-core/mds-utils'
 import { CreateJurisdictionType, JurisdictionDomainModel } from '../../@types'
 import { CreateJurisdictionsHandler } from './create-jurisdictions-handler'
 
 export const CreateJurisdictionHandler = async (
   jurisdiction: CreateJurisdictionType
-): Promise<ServiceResponse<JurisdictionDomainModel, ValidationError | ConflictError>> => {
+): Promise<ServiceResponse<JurisdictionDomainModel>> => {
   const [error, jurisdictions] = await CreateJurisdictionsHandler([jurisdiction])
-  return error || !jurisdictions ? ServiceError(error ?? new ServerError()) : ServiceResult(jurisdictions[0])
+  if (error) {
+    return ServiceError(error)
+  }
+  if (jurisdictions) {
+    return ServiceResult(jurisdictions[0])
+  }
+  return ServiceError({
```

Comment:
If I follow correctly, we're manually creating a ServiceErrorDescriptor here since we don't have a caught instance of Error?